src/Pages/Coursework/Fundamentals/scripts/4.Functions/E.Destructuring.js

```diff
@@ -5,32 +5,31 @@
 ===================================*/
 console.group("Functions: Destructuring");
 
-const readPerson = ({first, last, email, colors}) => {
+const readPerson = ({ first, last, email, colors }) => {
```

Comment:
some of this is just whitespace changes and quote style changes that can be ignored.

Really it was the missing commas and misspell of `groupEnd`